src/components/Graph/Graph.js

```diff
@@ -0,0 +1,264 @@
+        );
+      })
+    );
+  };
+
+  const renderYAxis = () => {
+    const yAxisValuesCount = 3;
+    const values = range(yAxisValuesCount).map(v => getYAxisValue((v + 1) / (yAxisValuesCount)));
+
+    return (
+      <ForeignObject width={width} height={height} x={graphPaddingHorizontal} y={0}>
+        <View style={{ flex: 1, backgroundColor: 'transparent' }}>
+          {values.map((v, i) => (
+            <BaseText
+              tiny
+              style={{ position: 'absolute', left: 4, top: getScreenY((i + 1) / (yAxisValuesCount)) }}
+              key={i}
+            >
+              {v}
+            </BaseText>
+          ))}
+        </View>
+      </ForeignObject>
+    );
+  };
+
+  const renderXAxis = () => {
+    const values = range(xAxisValuesCount).map(v => getXAxisValue(v / (xAxisValuesCount - 1)));
+    const valueWidth = 55;
+
+    return (
+      <ForeignObject y={height - 15}>
+        <View style={{ width, flexDirection: 'row', backgroundColor: 'transparent' }}>
+          {values.map((v, i) => (
+            <BaseText
+              style={{
+                width: valueWidth,
+                position: 'absolute',
+                left: ((i * ((graphWidth) / (xAxisValuesCount - 1))) - (valueWidth / 2)) + graphPaddingHorizontal,
+              }}
+              center
+              tiny
+              secondary
+              key={`${i}-${v}`}
+            >
+              {v}
+            </BaseText>
+          ))}
+        </View>
+      </ForeignObject>
+    );
+  };
+
+  const renderTooltip = (tooltipX: number, tooltipY: number) => {
+    const content = getTooltipContents(safeActiveDataPoint);
+
+    const tooltipWidth = 80;
+    const tooltipHeight = 40;
+    const tipHeight = 7;
+    const tipWidth = 10;
+    const borderRadius = 4;
+
+    return (
+      <>
+        <Path
+          fill={colors.graphPrimaryColor}
+          d={`
+            M ${tooltipX}, ${tooltipY}
+            l ${tipWidth / 2}, -${tipHeight}
+            h ${((tooltipWidth - tipWidth) / 2) - borderRadius}
+            a ${borderRadius}, ${borderRadius}, 0, 0, 0, ${borderRadius} -${borderRadius}
+            v -${tooltipHeight - (2 * borderRadius)}
+            a ${borderRadius}, ${borderRadius}, 0, 0, 0, -${borderRadius}, -${borderRadius}
+            h -${tooltipWidth - (2 * borderRadius)}
+            a ${borderRadius}, ${borderRadius}, 0, 0, 0, -${borderRadius}, ${borderRadius}
+            v ${tooltipHeight - (2 * borderRadius)}
+            a ${borderRadius}, ${borderRadius}, 0, 0, 0, ${borderRadius}, ${borderRadius}
+            h ${((tooltipWidth - tipWidth) / 2) - borderRadius}
+            l ${tipWidth / 2}, ${tipHeight}
+            z
+        `}
+        />
+        <ForeignObject
+          x={tooltipX - (tooltipWidth / 2)}
+          y={tooltipY - tooltipHeight - tipHeight}
+          width={tooltipWidth}
+          key={`${content}${extra || ''}`} // https://github.com/react-native-community/react-native-svg/issues/1357
```

Comment:
Please include a deeper explanation about which parts are affected by the bug and what is your workaround.